packages/botonic-nlu/src/preprocessing-tools/stemmer.ts

```diff
@@ -0,0 +1,40 @@
+// TODO: atm not using stemmers (returning default) as it will decrease accuracy with W.Embeddings
+// eslint-disable-next-line
+import * as PorterStemmer from 'natural/lib/natural/stemmers/porter_stemmer';
+// eslint-disable-next-line
+import * as PorterStemmerEs from 'natural/lib/natural/stemmers/porter_stemmer_es';
+// eslint-disable-next-line
+import * as PorterStemmerFr from 'natural/lib/natural/stemmers/porter_stemmer_fr';
+// eslint-disable-next-line
+import * as PorterStemmerIt from 'natural/lib/natural/stemmers/porter_stemmer_it';
+// eslint-disable-next-line
+import * as PorterStemmerPt from 'natural/lib/natural/stemmers/porter_stemmer_pt';
+// eslint-disable-next-line
+import * as PorterStemmerRu from 'natural/lib/natural/stemmers/porter_stemmer_ru';
+import { Language } from '../language';
+
+export class DefaultStemmer {
+  private _active: boolean;
+  constructor(active = false) {
```

Comment:
you can remove lines 17 till 20 with this
`constructor(private` active = false) {}